contracts/EVMScriptFactories/CSMSettleELStealingPenalty.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.6;
+
+import "../TrustedCaller.sol";
+import "../libraries/EVMScriptCreator.sol";
+import "../interfaces/IEVMScriptFactory.sol";
+import "../interfaces/ICSModule.sol";
+
+/// @author vgorkavenko
+/// @notice Creates EVMScript to settle EL stealing penalty for a specific node operators on CSM
+contract CSMSettleElStealingPenalty is TrustedCaller, IEVMScriptFactory {
+
+    // -------------
+    // ERRORS
+    // -------------
+
+    string private constant ERROR_EMPTY_NODE_OPERATORS_IDS =
```

Comment:
Why not custom errors?

Reply:
looks like there is some issue with brownie and custom errors checking, so let's leave it as is